docs/markdown/snippets/feature_force.md

```diff
@@ -0,0 +1,17 @@
+## Add `FeatureOption.force` to force a feature to enabled or disabled
+
+Sometimes we want to force a feature to be enabled or disabled, take for example
+when a feature should be automatically turned off for one platform, but off for
+another:
+
+```meson
+feat = get_option('feat')
```

Comment:
"feat: an achievement that requires great courage, skill, or strength."

Weird shortening if you ask me -- it derails my train of thought while trying to read this. Any reason not to just spell the whole thing out?